python/tests/unit/arcticdb/version_store/test_resample.py

```diff
@@ -90,6 +112,83 @@ def test_resampling_duplicated_index_value_on_segment_boundary(lmdb_version_stor
     )
 
 
+class TestResamplingBucketInsideSegment:
+
+    def test_all_buckets_have_values(self, lmdb_version_store_v1):
+        lib = lmdb_version_store_v1
+        sym = "test_inner_buckets_are_empty"
+        idx = pd.DatetimeIndex([
+            dt.datetime(2023, 12, 7, 23, 59, 47, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 48, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 49, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 50, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 51, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 52, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 53, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 55, 500000)
```

Comment:
Can we simplify the index values for all of these tests? It's easier to read if you can tell at a glance which bucket each value belongs in. e.g.
`idx = pd.DatetimeIndex([pd.Timestamp(i + 500, unit="ms") for i in range(8)])`